main.go

```diff
@@ -103,12 +103,14 @@ func main() {
 	var enableLeaderElection bool
 	var probeAddr string
 	var configPath string
+	var maxConcurrentReconciles int
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.StringVar(&configPath, "config", "", "Path to JSON file containing config")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.IntVar(&maxConcurrentReconciles, "MaxConcurrentReconciles", config.DefaultMaxConcurrentReconciles, "Maximum concurrent reconciles")
```

Comment:
@HumairAK  flag package is taking care of assigning the default value when there is no param. We are passing maxConcurrentreconciles (which will either have param value or default value to SetupWithManager function in the controller.